internal/config/pipeline.go

```diff
@@ -127,11 +127,14 @@ func (c *Config) PipelineProtos() ([]*pb.Pipeline, error) {
 			},
 		}
 
+		// TODO(briancain): Use the default ODR image for image_url so that it isn't required
+		// We do this already maybe in the ODR task launcher.
```

Comment:
Yeah, you can just make it empty and I think we should pick up the default that is configured with the server... that maybe won't work today but that's probably how it should work.